cypress/e2e/transfer.cy.ts

```diff
@@ -0,0 +1,74 @@
+import { transactionData } from '../support/constants';
+
+describe('Transfer', function () {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.interceptLoadTokenBalance();
+    cy.connectWallet();
+    cy.wait(1000); // wait tokens to be loaded
```

Comment:
Would rather have a more deterministic solution than wait, tends to cause unexpected fails

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/6/commits/71e5488e66fb445c3e40fbb529c76f8239f6cf15